packages/react-components/react-drawer/stories/Drawer/DrawerInline.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
```

Comment:
This storybook has an existing issue https://github.com/microsoft/fluentui/issues/29134 and it also happens for bottom drawer. Since it is not within the scope for this PR, the fix is not included. And I haven't figured out why.

Reply:
This was fixed yesterday. Should be merged soon.